### PR TITLE
fix: missing space in search

### PR DESCRIPTION
**Pull request by katrinan029**
## Description
**Problem**: The typeahead in search on the explore catalog has some minor formatting/spacing  issues in course and program titles. the first two words in the title are mashed together and are both made bold. Just makes for some potentially confusing reading. 

**Solution**: Add `margin-left` to create a space between matching results. 

| Before |
| ------------- |
| Screenshot 2023-05-30 at 11 55 15 AM|  

| After |
| ------------- |
|Screenshot 2023-05-30 at 11 51 16 AM|

Ticket Link
Link to the associated ticket
[Weird Formatting/Missing Spaces in Search on Explore Catalog](https://2u-internal.atlassian.net/browse/ENT-6984)

**Merge checklist:**
- [ ] Evaluate how your changes will impact existing consumers (e.g., `frontend-app-learner-portal-enterprise`, `frontend-app-admin-portal`, and `frontend-app-enterprise-public-catalog`). Will consumers safely be able to upgrade to this change without any breaking changes?
- [ ] Ensure your commit message follows the semantic-release conventional commit message format. If your changes include a breaking change, ensure your commit message is explicitly marked as a `BREAKING CHANGE` so the NPM package is released as such.
- [ ] Once CI is passing, verify the package versions that Lerna will increment to in the Github Action CI workflow logs.
    - *Note*: This may be found in the "Preview Updated Versions (dry run)" step in the Github Action CI workflow logs.

**Post merge:**
- [ ] Verify Lerna created a release commit (e.g., ``chore(release): publish``) that incremented versions in relevant package.json and CHANGELOG files, and created [Git tags](https://github.com/openedx/frontend-enterprise/tags) for those versions.
- [ ] Run the ``Publish from package.json`` Github Action [workflow](https://github.com/openedx/frontend-enterprise/actions/workflows/publish-from-package.yml) to publish these new package versions to NPM.
    - This may be triggered by clicking the "Run workflow" option for the ``master`` branch.
- [ ] Verify the new package versions were published to NPM (i.e., ``npm view <package_name> versions --json``).
    - *Note*: There may be a slight delay between when the workflow finished and when NPM reports the package version as being published. If it doesn't appear right away in the above command, try again in a few minutes.

### CODE REVIEW

File: packages/catalog-search/src/styles/_SearchSuggestions.scss [hunk 1]
```diff
@@ -29,6 +29,9 @@
   &:hover{
     color: white;
   }
+  &:not(:first-child) {
```
Comment (alex-sheehan-edx) [inline packages/catalog-search/src/styles/_SearchSuggestions.scss:32]: I thought it'd be something like this 😄

File: packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    renderWithRouter(<SearchSuggestionItem
+      url={mockData.course.url}
+      suggestionItemHandler={mockData.course.suggestionItemHandler}
+      hit={mockData.course.hit}
+      disableSuggestionRedirect={mockData.course.disableSuggestionRedirect}
+    />);
+    expect(screen.getByRole('link', { name: 'course catalog edX' })).not.toBeNull();
+    expect(screen.getByText('course')).not.toBeNull();
+    expect(screen.getByText('catalog')).not.toBeNull();
+    expect(screen.getByText('edX')).not.toBeNull();
```
Comment (alex-sheehan-edx) [inline packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx:32]: which edX text is this finding?

Comment (alex-sheehan-edx) [reply packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx:32]: is it the course key? If so then maybe assert the whole key

**Review by alex-sheehan-edx — APPROVED** [review]
I have a small question about your test but otherwise this is awesome! I would coordinate/talk to @macdiesel to learn more about releasing this.

File: packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx (hunk 2 above)
Comment (katrinan029) [reply packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx:32]: it's testing for the badge text so I believe it should only be `edX`

**Pull request merged**